portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,10 +47,12 @@ public Comment(String email, String comment, String date){
   }
   private DatastoreService datastore;
   private static final String COMMENT_DATE_FORMAT = "MMM dd,yyyy HH:mm";
```

Comment:
Nit: Move this static final above the non-static non-final variables

Reply:
Sure, is are there other rules similar to this one l should know about?

Reply:
If "similar" means "ordering of class variables", then "sort of". I think there's a specific defined ordering for the combination of [`private|public`, `static`, `final`], probably with public, static, and final at the top (with I think the order being the same as that as well).

If you mean more broadly "are the other formatting rules", then yes, there are:
https://google.github.io/styleguide/javaguide.html